nixos/modules/services/web-servers/phpfpm/default.nix

```diff
@@ -153,6 +154,20 @@ let
             details on configuration directives.
           '';
         };
+
+        systemdServiceName = mkOption {
+          type = types.str;
+          default = "phpfpm-${name}";
```

Comment:
I'm not sure if that's really needed: I'd be strongly against ever changing the name anyways, given it's wayy too much fallout for IMHO questionable gain (non-blocking).